static/js/src/advantage/react/utils/sendAnalyticsEvent.ts

```diff
@@ -0,0 +1,11 @@
+import { DataLayerEvent } from "globals";
+
+export const sendAnalyticsEvent = (
+  analyticsEvent: Omit<DataLayerEvent, "event">
+) => {
+  window.dataLayer &&
+    window.dataLayer.push({
```

Comment:
Just nitpicking. Could use optional chaining instead.
```suggestion
  window.dataLayer?.push({
```

Reply:
Thanks!